test/common/helm_install.go

```diff
@@ -0,0 +1,62 @@
+// Package common contains utility functions for all the tests in this repo
+package common
+
+import (
+	"crypto/tls"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+)
+
+// NamespaceName : common namespace variable to use in the tests
+var NamespaceName = "marklogic-" + strings.ToLower(random.UniqueId())
```

Comment:
We don't want to have a shared namespace. Each test should have it's own namespaced name to make sure the test e2e test is independant. Make the namespaceName as input parameter.